src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,56 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private static final int NOT_FOUND_INDEX = -1;
+    private int size = 0;
+    private final K[] keyArr;
+    private final V[] valArr;
+
+    public StorageImpl() {
+        this.keyArr = (K[]) new Object[MAX_SIZE];
+        this.valArr = (V[]) new Object[MAX_SIZE];
```

Comment:
redundant because there are no other variables with such names in scope of that method
```suggestion
        keyArr = (K[]) new Object[MAX_SIZE];
        valArr = (V[]) new Object[MAX_SIZE];
```